### PR TITLE
Side by side stacked bar description 22 1

### PR DESCRIPTION
**Pull request by AlexanderTemnov**

### CODE REVIEW

File: JSDemos/Demos/Charts/SideBySideStackedBar/description.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+## Customize Side-By-Side Stacked Bar Chart
+
+### Customize Chart Legend 
+
+Use the [verticalAlignment](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#verticalAlignment) and [horizontalAlignment](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#horizontalAlignment) properties of the legend object to specify the legend layout. 
+
+The [customizeLegend](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#customizeItems) function allows you to change the order, text, and visibility of legend items. In this demo, the **array.splice** method is used in the **customizeItems** function to sort the legend items. If you want to customize the legend's border, use properties collected in the [border](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/border) object.
```
Comment (AnnaKondratova) [inline JSDemos/Demos/Charts/SideBySideStackedBar/description.md:13]:
```suggestion
The [customizeLegend](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#customizeItems) function allows you to change the order, text, and visibility of legend items. In this demo, the **array.slice** method is used in the **customizeItems** function to sort the legend items. If you want to customize the legend's border, use properties collected in the [border](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/border) object. 
```

**Review by AnnaKondratova — DISMISSED** [review]

File: JSDemos/Demos/Charts/SideBySideStackedBar/description.md [hunk 2]
```diff
@@ -1 +1,23 @@
-In&nbsp;this example, side-by-side mode for the _stacked bar_ series type is&nbsp;used to&nbsp;group bar series based on&nbsp;their **stack** property value. Bar series with common characteristics are displayed and stacked in&nbsp;one column.
\ No newline at end of file
+This demo shows the _side-by-side stacked bar_ series that allows you to visualize data as bars stacked in different groups. This series type is useful when you compare an individual series' values with groups' total values.
```
Comment (arminal) [inline JSDemos/Demos/Charts/SideBySideStackedBar/description.md]:
```suggestion
This demo shows the _side-by-side stacked bar_ series that allows you to visualize data as bars stacked in different groups. This series type is useful when you compare an individual series' values with the total values of groups.
```

File: JSDemos/Demos/Charts/SideBySideStackedBar/description.md [hunk 3]
```diff
@@ -1 +1,23 @@
...
+
+### Customize Chart Legend 
+
+Use the [verticalAlignment](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#verticalAlignment) and [horizontalAlignment](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#horizontalAlignment) properties of the legend object to specify the legend layout. 
+
+The [customizeLegend](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#customizeItems) function allows you to change the order, text, and visibility of legend items. In this demo, the **array.splice** method is used in the **customizeItems** function to sort the legend items. If you want to customize the legend's border, use properties collected in the [border](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/border) object. 
+
+To distribute all legend items between several columns (in a vertically-oriented legend) or rows (in a horizontally-oriented legend), set the [columnCount](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#columnCount) or [rowCount](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#rowCount) property respectively.
```
Comment (arminal) [inline JSDemos/Demos/Charts/SideBySideStackedBar/description.md]: set the properties to what?

File: JSDemos/Demos/Charts/SideBySideStackedBar/description.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+
+### Configure Tooltips
+
+To configure tooltips in the chart, use the [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/) object. To enable the tooltips, assign **true** to the [enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property of this object. 
+
+### Export Chart
+
+To allow a user to export your side-by-side stacked bar chart into the PNG, JPEG, PDF, and SVG file or print the chart, set the [export.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/export/#enabled) property to **true**. In this demo, the exporting is enabled and you can click the "Exporting/Printing" button in the chart. This button invokes a drop-down menu with export and print commands.
```
Comment (arminal) [inline JSDemos/Demos/Charts/SideBySideStackedBar/description.md]:
```suggestion
To allow a user to export your side-by-side stacked bar chart into the PNG, JPEG, PDF, and SVG file or print the chart, set the [export.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/export/#enabled) property to **true**. In this demo, exporting is enabled and you can click the "Exporting/Printing" button in the chart. This button invokes a drop-down menu with export and print commands.
```

File: JSDemos/Demos/Charts/SideBySideStackedBar/description.md (hunk 3 above)
Comment (AlexanderTemnov) [reply JSDemos/Demos/Charts/SideBySideStackedBar/description.md]: @arminal  Let's change set to "specify" the properties

Comment (arminal) [reply JSDemos/Demos/Charts/SideBySideStackedBar/description.md]: yes

**Pull request closed without merging**